apps/dashboard/src/app/team/[team_slug]/[project_slug]/insight/components/BlueprintsPageHeader.tsx

```diff
@@ -0,0 +1,20 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import { PlusIcon } from "lucide-react";
+
+export function BlueprintsPageHeader() {
+  return (
+    <div className="flex grow flex-col">
+      <div className="border-border border-b py-10">
+        <div className="container flex items-center justify-between">
+          <h1 className="font-semibold text-3xl tracking-tight">Blueprints</h1>
+          <Button className="cursor-not-allowed gap-2 opacity-50" disabled>
+            <PlusIcon className="size-4" />
+            Create Blueprint (Coming Soon)
```

Comment:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yNOf1svJ8o3zjO7zQouZ/a917334f-ea03-4bbd-9df2-454800e34b9b.png)

This creates an Overflow on mobile

Reply:
fixed